src/components/content/catalog/services/update/UpdateService.tsx

```diff
@@ -31,33 +33,39 @@ import { getQueryKey } from '../query/useAvailableServiceTemplatesQuery';
 import UpdateResult from './UpdateResult';
 
 function UpdateService({
-    id,
+    serviceDetail,
     category,
     isViewDisabled,
-    registrationState,
-    isReviewInProgress,
 }: {
-    id: string;
+    serviceDetail: ServiceTemplateDetailVo;
     category: category;
     isViewDisabled: boolean;
-    registrationState: serviceTemplateRegistrationState;
-    isReviewInProgress: boolean;
 }): React.JSX.Element {
+    const [form] = Form.useForm();
     const ocl = useRef<Ocl | undefined>(undefined);
     const files = useRef<UploadFile[]>([]);
     const yamlValidationResult = useRef<string>('');
     const oclDisplayData = useRef<React.JSX.Element>(<></>);
     const updateResult = useRef<string[]>([]);
-    const serviceRegistrationStatus = useRef<serviceTemplateRegistrationState>(registrationState);
+    const serviceRegistrationStatus = useRef<serviceTemplateRegistrationState>(
+        serviceDetail.serviceTemplateRegistrationState as serviceTemplateRegistrationState
+    );
+    const isAvailableInCatalog = useRef<boolean>(serviceDetail.isAvailableInCatalog);
```

Comment:
would avoid storing all this here. We can directly take from this from backend response data.